examples/random-login/main.go

```diff
@@ -48,40 +48,39 @@ func provider() p.Provider {
 	})
 }
 
-// TODO: Deserialization does not yet work for external resources. Right now, it looks
-// like this structure is only implementable in typescript, but that will need to change.
```

Comment:
Re: @iwahbe's comment, do external resources still only work in TS or is this outdated?

If they're only partially working I'm less concerned about keeping an example around.

Reply:
There has been some work in pulumi/pulumi to support this, but I don't know its current extent.

Reply:
OK, I adjusted the component-provider example to use a resource reference for the resultant RandomPassword.